prow/cluster/jobs/all-periodics.yaml

```diff
@@ -68,6 +68,7 @@ periodics:
     nodeSelector:
       testing: test-pool
 
+- interval: 24h
```

Comment:
If you do this then delete the job `gcr_cleanup_istio_testing` above. The reason being, with this config the proxy-postsubmit-periodic overrode the above job. I noted this before and thought I opened an issue but I couldn't find it.

My concern being is we fix this bug and enable some job we don't want. We didn't have it for months and no one complained so I would assume that means it is not useful

Reply:
Is that the semantic of the yaml parser, it overrides in the case of conflicts? If so, I agree that it'd be better to delete the overriden job.

Reply:
Done.

Reply:
Not sure, I just checked prow.istio.io and saw only the proxy postsubmit one has run, not the gcr_cleanup

Reply:
https://prow.istio.io/?job=gcr_cleanup_istio_testing has never been ran, so it must be overriden